src/run.rs

```diff
@@ -64,29 +64,28 @@ pub async fn run(matches: &ArgMatches) -> Result<(), AocError> {
         .chain(std::iter::once(input));
 
     let cmd = cmd("cargo", args);
-    let reader = cmd.dir(dir).stderr_to_stdout().reader()?;
+    let reader = cmd.dir(&dir).stdout_capture().stderr_null().reader()?;
```

Comment:
Need to get both stderr and stdout, but we need a way to also only get stdout for asserting what the answers will be with the regexes